examples/latex.jl

```diff
@@ -15,13 +14,14 @@ function main(window)
              textinput() >>> texᵗ,
              hbox("Show as a block", hskip(1em), checkbox(mode) >>> modeᵗ),
              vskip(1em),
-             tex(t, block=mode)) |> pad(1em) |> maxwidth(30em)
+             tex(t, block=mode)) |> Escher.pad(1em) |> maxwidth(30em)
```

Comment:
Qualifying it always is probably easier for new users who might get confused by the warning appearing only in some cases!